### DISCUSSION TITLE
Use RealmA to authenticate to RealmB

### DISCUSSION THREAD

**Discussion by michelangelo136** [keycloak/keycloak#31059] [Q&A]
We had an issue with LDAP timing out and making it impossible to implement an integration between Netbird and Keycloak while using LDAP for the user federation.

The workaround is to use a new realm just for Netbird and authenticate the users on the existing realm, how can I configure the RealmB to authenticate the users using the authentication provider Keycloak from RealmA? is it also posible to import all the users in a specific group to RealmB?

**Comment by Xulunix** [comment]
Yes, you can do that. 
Add an OIDC Identity provider to RealmB with a client in RealmA as target.
You can set groups via mappers

**Reply by michelangelo136** [reply]
Thank you for the reply, do you have any guide/reference available? what option should I enable on the RealmA client?

I get the error "Unexpected error when authenticating with identity provider", error="identity_provider_login_failure", and userId="null"
